src/index.html

```diff
@@ -10,13 +10,106 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
```

Comment:
HTML Formatting: The `link` tag has two attributes on one line. For better readability, it's recommended to start each attribute on a new line with a 2-space indentation.